<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoleFrame</class>
 <widget class="QWidget" name="RoleFrame">
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>163</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="roleLabel0"/>
   </item>
   <item>
    <widget class="QLabel" name="roleLabel1"/>
   </item>
   <item>
    <widget class="QLabel" name="roleLabel2"/>
   </item>
   <item>
    <widget class="QLabel" name="roleLabel3"/>
   </item>
   <item>
    <widget class="QLabel" name="roleLabel4"/>
   </item>
   <item>
    <widget class="QLabel" name="roleLabel5"/>
   </item>
   <item>
    <widget class="QLabel" name="roleLabel6"/>
   </item>
   <item>
    <widget class="QLabel" name="roleLabel7"/>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>163</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
